### PR TITLE
Add super key gala shortcut to tooltip

### PR DESCRIPTION
**Pull request by zeebok**
Addresses #184

### CODE REVIEW

**Review by danrabbit — COMMENTED** [review]
It looks like Granite doesn't really know how to handle adding just `<super>` to accels. It gives you an extra `+`. But that would probably be the best way to do this to cut down on repeat localization.

I would be okay merging this, but I think it would be better to fix the granite bug and then add `<super>` to accels so we get the correct localization for RTL and with delimiters etc

**Comment by zeebok** [comment]
Alright, I will take a look at Granite and get that fixed. If I have any questions for Granite, where is the best place to post?

**Comment by danrabbit** [comment]
@zeebok sent you an invite link to the contribute slack in DM on the community slack :)

File: src/Indicator.vala [hunk 1]
```diff
@@ -114,6 +128,12 @@ public class Slingshot.Indicator : Wingpanel.Indicator {
         }
 
         indicator_grid.tooltip_markup = Granite.markup_accel_tooltip (accels, _("Open and search apps"));
+
+        if (gala_behavior_settings != null) {
+            if ("wingpanel" in gala_behavior_settings.get_string ("overlay-action")) {
+                indicator_grid.tooltip_markup += ", ⌘";
+            }
+        }
```
Comment (danrabbit) [inline src/Indicator.vala:136]: Thanks for fixing the granite util! Now we can add `<Super>` to accels instead of manually adding to the string

**Review by danrabbit — CHANGES_REQUESTED** [review]

File: src/Indicator.vala (hunk 1 above)
Comment (zeebok) [reply src/Indicator.vala]: Yep, feels much cleaner!

**Review by danrabbit — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**